working-with-fhir-igs/hl7v2-to-dk-medcom/README.md

```diff
@@ -0,0 +1,156 @@
+# Ballerina HL7v2 to Danish FHIR Converter
+
+This project demonstrates how to use Ballerina's HL7v2 to FHIR conversion utilities along with the MedCom240 Ballerina library to convert standard HL7v2 messages into Danish FHIR resources (referring MedCom Implementation Guide version 2.4.0.)
+
+## Overview
```

Comment:
Let's add a diagram for the flow

Reply:
fixed in [4a378ec](https://github.com/ballerina-guides/healthcare-samples/pull/38/commits/4a378ec4f64c338c69ab412bcd053c7d95901093)